### PR TITLE
Update text & graph data

### PR DESCRIPTION
**Pull request by Calemy**
Changed graph data and text to make people understand stuff better.

[88a9adb](https://github.com/yo-ru/beatmap.download/commit/1842c6ce0abd9216ad91a0ea2b1685391920e7d6) Needs some styling on the new button but everything else should be fine. It has also been tested throughly on local machine.

Images:
[new status & button](https://x.catboy.best/oOuEjR5.png)
[new graph name & larger dataset](https://x.catboy.best/VrIxbcE.png)

### CODE REVIEW

**Comment by HorizonCode** [comment]
imo "DOWN" "UP" and so on where fine

**Comment by HorizonCode** [comment]
and the switcher button looks a bit misplaced at the top imo

**Comment by Calemy** [comment]
> imo "DOWN" "UP" and so on where fine

for me it looked like a lot of wasted space, that's (for most part) personal preference

> and the switcher button looks a bit misplaced at the top imo

that's likely because it's not properly styled yet. I just think it's just easier to see than on the complete bottom of a page pretty much "semi-hidden". Hence why i mentioned it needs to get proper styling first.

**Review by yo-ru — CHANGES_REQUESTED** [review]
- <br> is generally not good practice.
- The button will remain in the footer for now until I move it.

File: static/css/style.css [hunk 1]
```diff
@@ -135,6 +135,10 @@ div.header>p.caption {
   margin: 0;
 }
 
+div.region {
+  margin: 0 0 2em 0;
+}
+
```
Comment (yo-ru) [inline static/css/style.css:141]:
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -24,23 +24,28 @@
...
             <div id="search-chart" class="chart-style"></div>
-            <div id="download-chart" class="chart-style"></div>
             <div id="downloadtime-chart" class="chart-style"></div>
         </div>
     </div>
     <div class="footer animate__animated animate__fadeIn animate__delay-1s">
-        Created with <span style="color: #ffaeae">&#10084;</span> by Yoru, Lemres, & HorizonCode | Powered by Kagenko | <a id="serverSwitch">Switch to  <span class="fi fi-eu"></span></a>
+        Created with <span style="color: #ffaeae">&#10084;</span> by Yoru & Lemres | Powered by Kagenko
+        <br>
+        Important contributors: HorizonCode
```
Comment (yo-ru) [inline index.html:43]:
```suggestion
        Created with <span style="color: #ffaeae">&#10084;</span> by Yoru & Lemres | Powered by Kagenko

```

**Review by yo-ru — CHANGES_REQUESTED** [review]
Please take a look over these changes and approve, debate, & deny them accordingly.

File: static/js/script.js [hunk 3]
```diff
@@ -61,21 +61,24 @@ const buildPage = async (apiUrl) => {
...
         degradedEndpoints.classList.add('degraded')
-        if (up[0] === 0) degradedEndpoints.innerHTML += '<span>Status</span>'
-        if (up[1] === 0) degradedEndpoints.innerHTML += '<span>Search</span>'
-        if (up[2] === 0) degradedEndpoints.innerHTML += '<span>Download</span>'
+        let down = []
+        degradedEndpoints.innerHTML += '<span>'
+        for(let i = 0; i < up.length; i++) {
+          if(up[i] === 0) down.push(i == 0 ? "Status" : i == 1 ? "Search" : "Download")
+        }
+        degradedEndpoints.innerHTML += `${down.join(" & ")} down</span>`
```
[the remark this answers is not in the document]
Comment (Calemy) [reply static/js/script.js]: personally would leave the down part to make sure people understand that those routes are actually down.

File: index.html [hunk 4]
```diff
@@ -24,23 +24,27 @@
             <h1 class="title">beatmap.download</h1>
             <p class="caption">a osu! beatmap mirror list</p>
         </div>
+        <div class="region">
+            <button id="serverSwitch" class="btn">
+                Switch to <span class="fi fi-eu"></span>
+            </button>
+        </div>
```
Comment (yo-ru) [inline index.html:31]:
```suggestion
```

File: index.html [hunk 5]
```diff
@@ -24,23 +24,27 @@
...
             <div id="uptime-chart" class="chart-style"></div>
             <div id="status-chart" class="chart-style"></div>
             <div id="search-chart" class="chart-style"></div>
-            <div id="download-chart" class="chart-style"></div>
             <div id="downloadtime-chart" class="chart-style"></div>
         </div>
     </div>
     <div class="footer animate__animated animate__fadeIn animate__delay-1s">
-        Created with <span style="color: #ffaeae">&#10084;</span> by Yoru, Lemres, & HorizonCode | Powered by Kagenko | <a id="serverSwitch">Switch to  <span class="fi fi-eu"></span></a>
+        Created with <span style="color: #ffaeae">&#10084;</span> by Yoru & Lemres | Powered by Kagenko
```
Comment (yo-ru) [inline index.html:41]:
```suggestion
        Created with <span style="color: #ffaeae">&#10084;</span> by Yoru, Lemres, & HorizonCode | Powered by Kagenko | <a id="serverSwitch">Switch to <span class="fi fi-eu"></span></a>

```

**Review by yo-ru — CHANGES_REQUESTED** [review]
more changes.

**Review by yo-ru — APPROVED** [review]

**Pull request merged**